docs/overview/Introduction.md

```diff
@@ -28,9 +28,47 @@ A CentrifugeID has the following credentials:
 
 * Ethereuem Accounts: When interacting with a smart contract on Ethereum, an account needs to be linked to the identity to act on its behalf. The linked Ethereum accounts are the accounts that are allowed to interact with DApps utilizing Centrifuge OS.
 
+Centrifuge OS users register their identity on the public Ethereum network and
+maintain a set of public keys as part of their identity. These identities and their keys are used to resolve nodes on the P2P layer when one node wants to transmit data to
+another Centrifuge user. This means that nodes on the P2P layer find others by
+looking up their public keys from the public identity register based on CentrifugeID.
+
+## Centrifuge Peer-to-Peer Network
+
+A peer-to-peer (P2P) network consists of two or more systems connected and sharing resources without going through a separate server. Centrifuge consists of a P2P network for decentralized data exchange using smart contracts on Ethereum. The Centrifuge P2P node supports handling of incoming and outgoing data using a service bus principal where plugins and outside systems can subscribe to messages about specific objects.
```

Comment:
I wouldn't say "two or more". We have more than two nodes connected at any time. 

These two sentences are a bit repetitive: 
A peer-to-peer (P2P) network consists of two or more systems connected and sharing resources without going through a separate server. Centrifuge consists of a P2P network for decentralized data exchange using smart contracts on Ethereum. 

I wouldn't use the term "service bus". We don't use that anywhere.